in_toto/in_toto_sign.py

```diff
@@ -316,6 +327,13 @@ def main():
       "Path(s) to PEM formatted key file(s), used to sign the passed link or"
       " layout metadata or to verify its signatures."))
 
+  parser.add_argument("-t", "--key-type", dest="key_type",
+      type=str, choices=in_toto.util.SUPPORTED_KEY_TYPES,
+      nargs="+", help=(
+      "Specify the key-type of the keys specified by the '--key'"
```

Comment:
Some users might misinterpret the help description to mean that one single key type is expected for one or more keys.

Is the following invocation of in-toto-sign supported?  I'm not sure the help description makes it clear enough that it is.
```
$ in-toto-sign -k key1 key2 key3 -t rsa ed25519 rsa
```